src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTest.cs

```diff
@@ -969,21 +968,24 @@ public async Task ReadAsStreamAsync_HandlerProducesWellBehavedResponseStream(boo
                         // Boolean properties returning correct values
                         Assert.True(responseStream.CanRead);
                         Assert.False(responseStream.CanWrite);
-                        Assert.False(responseStream.CanSeek);
+                        Assert.Equal(PlatformDetection.IsBrowser, responseStream.CanSeek);
```

Comment:
Do we want this check after all? Hypothetically, when there's new platform with response stream that can seek as well, this line would need an update again. Or does it has some other purpose here?